lib/tasks/zipcodes.rake

```diff
@@ -0,0 +1,164 @@
+require 'open-uri'
```

Comment:
This script is provided by `my_zipcode_gem`. We _could_ download the data ourselves, put it in our seeds folder, and skip this script altogether. Thoughts? (Only `all_us_zipcodes.csv` is large, and it's still less than 2 MiB.)